test/t_web/channels/feed_channel_test.exs

```diff
@@ -390,7 +379,6 @@ defmodule TWeb.FeedChannelTest do
       assert reply == %{
                "mode" => "normal",
                "since_live_date" => since_live_date,
-               "match_expiration_duration" => 172_800 = Matches.pre_voicemail_ttl(),
```

Comment:
just a note, I know this field needs to be removed but I believe it's best to make these changes in separate PRs otherwise it's easy to overlook ...